src/main/resources/db/migration/V1_3__add_header_tables.sql

```diff
@@ -0,0 +1,24 @@
+create table email_header
+(
+    email_id   varchar(255),
+    id         varchar(255) not null,
+    header_key enum ('IN_REPLY_TO','REFERENCES','MESSAGE_ID'),
+    primary key (id)
+) engine = InnoDB;
+
+
+create table email_header_value
+(
+    header_id varchar(255) not null,
+    value     varchar(255)
+) engine = InnoDB;
```

Comment:
A list of message-ids can be a lot longer than 255 characters, might want to use another datatype.